shared/src/commonMain/kotlin/co/nimblehq/kmm/template/data/repository/RepositoryImpl.kt

```diff
@@ -0,0 +1,16 @@
+package co.nimblehq.kmm.template.data.repository
```

Comment:
please keep package name in plural 🙏 

```suggestion
package co.nimblehq.kmm.template.data.repositories
```